### PR TITLE
fix(wasm): prevent panic in the wasm

### PR DESCRIPTION
**Pull request by erka**
Fixes: #377

### CODE REVIEW

File: flipt-client-browser/src/index.ts [hunk 1]
```diff
@@ -100,7 +100,8 @@ export class FliptEvaluationClient {
 
     const data = await resp.json();
 
-    const engine = new Engine(namespace, data);
+    const engine = new Engine(namespace);
+    engine.snapshot(data);
```
Comment (markphelps) [inline flipt-client-browser/src/index.ts:104]: we'll have to do the same thing in the node SDK now as well I think

Comment (erka) [reply flipt-client-browser/src/index.ts:104]: I missed that. Ty!

**Review by markphelps — APPROVED** [review]
looks great, ty!

**Comment by erka** [comment]
@markphelps does kodiakhq just ignore status checks? My commit wasn't signed

**Comment by markphelps** [comment]
@erka ah sorry. I think the DCO check wasn't required (i fixed it now) so kodiak merged it

**Pull request merged**